src/errors/NoDataException.java

```diff
@@ -0,0 +1,7 @@
+package errors;
+
+public class NoDataException extends CommandException {
```

Comment:
Usually we create multiple exceptions if we need to handle them differently. It is not our case.
We handle all these exceptions in the same way. And they have same meaning - config argument is incorrect.
That is why it is better to keep only CommandException. 
Instead of `NoDataException` use  `new CommandException("No data type defined!")`
Similar for the other exceptions